src/index.css

```diff
@@ -1,13 +1,233 @@
-body {
+:root {
+  --PRIMARY: rgb(255, 255, 255);
+  /* --secondary-color:; */
+  --ACCENT-COLOR: #fab75f;
+  --ACCENT-COLOR-LIGHT: #fed7aa;
+  --GREY: #6c757d;
+  --GREY-DARK: #343a40;
+  --PURE-BLACK: #212529;
+  --PURE-WHITE: #ffffff;
+  --BREAKPOINT-SML: 576px;
+  --BREAKPOINT-MED: 768px;
+  --BREAKPOINT-LRG: 992px;
+  --BREAKPOINT-XL: 1200px;
+  --GREY-LIGHT: #D3D3D3;
+  --SLATE-50: #f8fafc;
+  --STONE-100: #F5F5F4;
+  /* colors required for the footer */
+  --footer-bg: #343a40;
+  --color: #f2f3fb;
+  --color-smooth: #9da2ae;
+  --input-border: #3a456a;
+  --btn-bg-color: #f1f2fc;
+  --btn-hover-bg: #d5d9f6;
+  --btn-color: #1e2641;
+}
+
+/* default and resetting */
+
+li {
+  list-style: none;
+}
+
+a {
+  text-decoration: none;
+  color:var(--PURE-BLACK);
+}
+::selection {
+  background-color: var(--ICONS-ORANGE);
+  color: white;
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+
```

Comment:
remove the space here

Reply:
oh this is just for now It will be removed in the final push